simulators/portal-interop/src/main.rs

```diff
@@ -206,80 +199,163 @@ dyn_async! {
                 }
             ).await;
 
-            // Test find content receipts over uTP
+            // Test find content Header With Proof
             test.run(
                 TwoClientTestSpec {
-                    name: format!("FIND_CONTENT Receipts over uTP {} --> {}", client_a.name, client_b.name),
+                    name: format!("FIND_CONTENT Header block 1 {} --> {}", client_a.name, client_b.name),
```

Comment:
Same nitpick as for https://github.com/ethereum/portal-hive/pull/100#discussion_r1321085720

```suggestion
                    name: format!("FindContent Block Header: block number 1 {} --> {}", client_a.name, client_b.name),
```

Reply:
resolved